src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/script.js.template

```diff
@@ -232,6 +232,20 @@ document.addEventListener("readystatechange", (e) => {
 });
 document.addEventListener("DOMContentLoaded", function(e) {
     setTopMargin();
+    // Reset animation for type parameter target highlight
+    document.querySelectorAll("a").forEach((link) => {
+        link.addEventListener("click", (e) => {
+            const href = e.currentTarget.getAttribute("href");
+            if (href && href.startsWith("#") && href.indexOf("type-param-") > -1) {
```

Comment:
Does the `href.startsWith("#")` mean that this only works for references within the same file? Is it even a sensible question to wonder if you can link to type parameters in other files?

Reply:
This is  a workaround to allow the highlight animation to play multiple times if the type parameter link is clicked repeatedly. This is only necessary for internal links and does not affect links to other files. 

Examples for a type parameter link in another class are the links to the type parameters of `Map.Entry` in the [nested class summary](https://cr.openjdk.org/~hannesw/8313931/api.02/java.base/java/util/Map.html#nested-class-summary) of `java.util.Map`, or the links back to `Map` type parameters in the [enclosing interface definition](https://cr.openjdk.org/~hannesw/8313931/api.02/java.base/java/util/Map.Entry.html) in `Map.Entry`.

Reply:
OK, understood